src/openzaak/components/zaken/api/viewsets.py

```diff
@@ -660,6 +657,7 @@ def perform_create(self, serializer):
             else None,
             gezetdoor=str(status_obj.gezetdoor) if status_obj.gezetdoor else None,
         )
+        send_zaak_cloudevent(ZAAK_GEMUTEERD, zaak, self.request)
```

Comment:
This is already done via `CloudEventCreateMixin` on the viewset, so you can leave this out
```suggestion
```